src/commands/channels/create.ts

```diff
@@ -83,7 +83,7 @@ the command currently requires that both the
   tryDomainOptionsValidation(options: any, domainOptions: DomainOption[]) {
     super.tryDomainOptionsValidation(options, domainOptions)
 
-    validateChannelName(options.name)
+    validateChannelName(options.name, '--name')
```

Comment:
Let's pass the name of the flag without the "--". Also applies to line 54 below.